platform/uboot/boot.txt

```diff
@@ -1,16 +1,17 @@
 setenv knladdr 0x40008000
-setenv dtbaddr ${fdt_addr_r}
-setenv loadaddr ${kernel_addr_r}
+setenv dtbaddr 0x5fa00000
+setenv loadaddr 0x50008000
 setenv dtboaddr 0x52008000
 
 setenv bootargs "
  init=/init rootwait ro androidboot.boottime=223.708 androidboot.selinux=permissive
  androidboot.revision=2.0 androidboot.board_id=0x1234567 init_time=1552948124 androidboot.serialno=${serial#}
- androidboot.slot_suffix=_a firmware_class.path=/vendor/etc/firmware video=HDMI-A-1:e ${debug_bootargs}
+ androidboot.slot_suffix=_a firmware_class.path=/vendor/etc/firmware of_devlink=0 ${debug_bootargs}
```

Comment:
boots fine on opi3 without `of_devlink=0` with 5.6-rc4 kernel

Reply:
1. I want to keep 5.4 for some time now.
2. I think this is a feature that is only introduced in Android kernel.

Reply:
I use 5.6-rc4 from Google (android-mainline branch)
I need 5.6 to enable GPU on opi3.
We can keep both and select required by makefile

Reply:
I upgraded to 5.6-rc kernel, and it still fails to boot w/o of_devlink=0.

Maybe some driver of A64 is not adapted to of_devlink.

Reply:
I've added this argument only for pinetab, could you test it?
https://github.com/glodroid/glodroid_device/commit/07cec8f4d1c419c5e8c538d936fb0918fbbabdf3

Reply:
It's okay.